### PR TITLE
Run3 XSec Updates

### PR DESCRIPTION
**Pull request by MatthewDittrich**
adding more run 3 xsec values and references (along with some initial 2023 edits)

### CODE REVIEW

File: topcoffea/params/xsec.yml [hunk 1]
```diff
@@ -131,26 +131,47 @@ SSWW   : 0.02794
...
 GluGlutoContinto2Zto4E_13p6TeV         : 0.00305851
 GluGlutoContinto2Zto4Tau_13p6TeV       : 0.00303575
 
+# Numbers from SMP-22-017
+TbarWplus_DR_AtLeastOneLepton_13p6TeV  : 24.2
+TWminus_DR_AtLeastOneLepton_13p6TeV    : 24.2
+
 #Numbers from SMP-24-005
-VH_HtoNonbb_13p6TeV                    : 0.9561
+VH_HtoNonbb_13p6TeV                    : 2.55
```
Comment (bryates) [inline topcoffea/params/xsec.yml:163]: This one is quite different from the original value. Was the value of `0.9561` just incorrect?

Comment (bryates) [reply topcoffea/params/xsec.yml:163]: Ok, thanks for confirming.

**Comment by MatthewDittrich** [comment]
This is all the xsecs for Run 3 that I was wanting to add. I have also added some preliminary 2023 things

File: topcoffea/params/xsec.yml [hunk 2]
```diff
@@ -127,51 +127,49 @@ tWll : 0.01123
...
-
-# Obatined Using GenXSecAnalyzer
-ZH_ZtoAll_Hto2Wto2L2Nu_M-125_13p6TeV   : 0.0195
-GluGluZHToHtoWWto2L2Nu_13p6TeV         : 0.0033
+GluGluToContinto2Zto2E2Mu_13p6TeV      : 0.0061150
+GluGluToContinto2Zto2E2Tau_13p6TeV     : 0.0061150
+GluGluToContinto2Zto2Mu2Tau_13p6TeV    : 0.0061150
+GluGlutoContinto2Zto4Mu_13p6TeV        : 0.003
+GluGlutoContinto2Zto4E_13p6TeV         : 0.003
+GluGlutoContinto2Zto4Tau_13p6TeV       : 0.003
```
[the remark this answers is not in the document]
Comment (MatthewDittrich) [reply topcoffea/params/xsec.yml:161]: Yes, this is the most up-to-date from the twiki I obtained them from

File: topcoffea/params/params.json [hunk 3]
```diff
@@ -4,8 +4,10 @@
     "lumi_2016": 16.81,
     "lumi_2017": 41.48,
     "lumi_2018": 59.83,
-    "lumi_2022": 8.07696,
+    "lumi_2022": 7.9804,
```
[the remark this answers is not in the document]
Comment (MatthewDittrich) [reply topcoffea/params/params.json:7]: Era B is not recommended for analysis work (even though some other analyses are using it lol), so I dropped the era B lumi

**Review by bryates — APPROVED** [review]
@MatthewDittrich thanks for addressing my comments. I'm happy with this PR, so once @kmohrman reviews it, we can merge!

File: topcoffea/data/goldenJsons/Cert_Collisions2023_366442_370790_Golden.txt [hunk 4]
```diff
@@ -0,0 +1,312 @@
+{
```
Comment (kmohrman) [inline topcoffea/data/goldenJsons/Cert_Collisions2023_366442_370790_Golden.txt:1]: Can you modify the readme to document where the 2023 golden json comes from?

**Comment by kmohrman** [comment]
Looks good to me, thanks Matthew. And thanks Brent for the review

**Pull request merged**